src/deploy/functions/release/planner.ts

```diff
@@ -10,39 +10,63 @@ export interface EndpointUpdate {
   deleteAndRecreate?: backend.Endpoint;
 }
 
-export interface RegionalChanges {
+export interface Changeset {
   endpointsToCreate: backend.Endpoint[];
   endpointsToUpdate: EndpointUpdate[];
   endpointsToDelete: backend.Endpoint[];
 }
 
-export type DeploymentPlan = Record<string, RegionalChanges>;
+export type DeploymentPlan = Record<string, Changeset>;
 
 export interface Options {
   filters?: string[][];
   // If set to false, will delete only functions that are managed by firebase
   deleteAll?: boolean;
 }
 
-/** Calculate the changes needed for a given region. */
+/** Calculate the changesets needed for a given region by grouping endpoints with keyFn. */
 export function calculateRegionalChanges(
```

Comment:
We ought to rename this to something like "calculateChangesets" now that it is not specifically for regions. There isn't really any specific logic for grouping by region baked into this any more